pyproject.toml

```diff
@@ -301,7 +301,7 @@ git = "https://github.com/RasaHQ/pytest-sanic"
 branch = "fix_signal_issue"
 
 [tool.poetry.group.dev.dependencies]
-ruff = ">=0.0.255,<0.0.256"
+ruff = ">=0.0.255,<0.0.277"
```

Comment:
Please fix any new errors found by `ruff` as part of this upgrade.

Reply:
Ruff upgrade opens a whole new can of worms,
```
Found 68 errors.
[*] 53 potentially fixable with the --fix option.
```

I think that we can skip upgrading ruff in this PR and instead create a ticket to do it later